how-we-work-together.md

```diff
@@ -74,7 +74,7 @@ No other person has as broad and nuanced a project view as the project manager (
 

```

Comment:
This is a list of action that a PM should take (help the point of contact, protect time), but this is just a statement of what should be done. Can this be rephrased to what a PM should do about this issue? Are they the ones who should correct the client? Should they consult someone about changing the timeline, scope or budget?